source/Dig/base.util.scriptexpression_ng.bmx

```diff
@@ -1603,15 +1603,22 @@ Function SEFN_UCFirst:SToken(params:STokenGroup Var, context:SScriptExpressionCo
 End Function
 
 
-Function SEFN_Hour:SToken(params:STokenGroup Var, context:SScriptExpressionContext var)
-	'Print params.reveal("PARAMS: "+ params.added)
-	'DebugStop
-	Local first:SToken = params.GetToken(0)
-	If params.added > 1 Then Return New SToken( TK_BOOLEAN, False, first.linenum, first.linepos )
-	Local Hour:String = CurrentDate( "%H" )
-	Return New SToken( TK_NUMBER, Int(hour), first.linenum, first.linepos )
+Function SEFN_Csv:SToken(params:STokenGroup Var, context:SScriptExpressionContext var)
+	Local values:TStringBuilder = New TStringBuilder(params.GetToken(1).GetValueText())
+	Local index:Int = Int(params.GetToken(2).GetValueText())
+	Local separator:String = ","
```

Comment:
Ich hatte überlegt, ob Semikolon als Default sinnvoller wäre, da ein Komma in Snippets, aus denen man Texte zusammenbauen möchte, häufiger vorkommen könnten. Aber in den (hoffentlich wenigen) Fällen kann man dann den optionalen Separator mitgeben.

Reply:
Ja ich kann gern auf das Semikolon umstellen.